.github/workflows/monitor.yaml

```diff
@@ -24,6 +25,8 @@ jobs:
       - uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install planemo
```

Comment:
All pip calls are now removed from the monito script and planemo is now installed here which should respect `PLANEMO_VERSION` or?

Reply:
👍  I think I just looked at the commit, not the PR.